dpub/output.py

```diff
@@ -26,14 +26,19 @@ def compose(test, mode, max=_CELL_MAX_CHARS):
     if mode == 'message':
         # Every message is an element in the values array
         for it in test.items:
+            if (test.update == "New") and test.id:
+                _append(values, test.id, max)
```

Comment:
but this would write the test id in the column just before the trace. That's not always correct. Tests could be defined in any other different columns. Even in another different sheet within the same document. I think that the approach should be:

1.- write the tests identified that are at the same time in the spreadsheet and in the newman collection (it's happening already)
2.- the remaining tests in the parsed items array are new tests. You know it. Otherwise they wouldn't be there
3.- Calculate the last read test row. For example, if last test definition is in cell A15. You know that in cell A16 there's nothing and is the point to start adding new tests
4.- We can see if we got test name besides test identifier. In a first approach, we can write the name on the next right cell to the identifier (if test id is written in A16, test name is suitable to be in B16)
5.- Move on by rows until remaining new tests are completed

The hardest stuff here is the point 3: calculate where to write the new tests. To approaches: the first is the one that i mentioned about knowing from where has been read the last defined test. The second is probably easier; when writing the existing tests, the rows cursor moves ahead n times. The initial read test + n should be the position where start writing the new tests

Reply:
I've done big changes following this comments and logic, over this new branch: feat/adding_new_tests